### PR TITLE
Add Problems link

### LINKED ISSUE #7: add url to original problem on problems
every solution should have a link to the problem as comment in the first line

### PR DESCRIPTION
**Pull request by benildacoari**
this closes #7

### CODE REVIEW

File: coj/4265.cpp [hunk 1]
```diff
@@ -1,4 +1,5 @@
-#include <bits/stdc++.h>
+// Problem link: https://coj.uci.cu/24h/problem.xhtml?pid=4265
+#include <bits/stdc++.h>
```
Comment (qf-jonathan) [inline coj/4265.cpp]: many files have this weird character, should remove it

File: kattis/diagonalcut.cpp [hunk 2]
```diff
@@ -1,4 +1,5 @@
-#include <bits/stdc++.h>
+// Problem link: https://open.kattis.com/problems/diagonalcut
+#include <bits/stdc++.h>
```
Comment (qf-jonathan) [inline kattis/diagonalcut.cpp]: same here

**Review by qf-jonathan — CHANGES_REQUESTED** [review]
there is a weird character on files, should change it first

File: coj/4265.cpp (hunk 1 above)
Comment (benildacoari) [reply coj/4265.cpp]: solved

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "workbench.tree.indent": 20
+}
```
Comment (qf-jonathan) [inline .vscode/settings.json]: vscode settings file should not be part of the PR

**Review by qf-jonathan — CHANGES_REQUESTED** [review]
remove vscode setting file

File: .vscode/settings.json (hunk 3 above)
Comment (benildacoari) [reply .vscode/settings.json]: done

**Review by qf-jonathan — APPROVED** [review]
great!
